packages/react-signer/src/Modal.tsx

```diff
@@ -183,25 +206,15 @@ class Signer extends React.PureComponent<Props, State> {
     const { isExternal, isHardware, hardwareType } = extractExternal(currentItem.accountId);
 
     return (
-      <Modal.Actions
-        onCancel={
-          isQrVisible
-            ? this.onCancelQr
-            : this.onCancel
-        }
-      >
-        {!isRenderError && (!isQrVisible || !isQrScanning) && (
+      <Modal.Actions cancelLabel={signedTx ? t('Close') : undefined} withOr={!signedTx} onCancel={isQrVisible ? this.onCancelQr : signedTx ? this.onCancelSign : this.onCancel}>
+        {!isRenderError && (!isQrVisible || !isQrScanning) && !signedTx && (
           <>
             <Button.Or />
             <Button
-              className='ui--signer-Signer-Submit'
+              className="ui--signer-Signer-Submit"
```

Comment:
```suggestion
              className='ui--signer-Signer-Submit'
```

actually not sure why eslint doesn't complain about " vs '

Reply:
I remember there's an exception to the `'` vs `"` which is this exact case - plain strings as classNames. I'll see if I can force my eslint to use single quotes everywhere.